.github/workflows/integration-tests.yml

```diff
@@ -358,6 +446,11 @@ jobs:
           this-job-name: ETH Smoke Tests ${{ matrix.product.name }}
           test-results-file: '{"testType":"go","filePath":"/tmp/gotest.log"}'
         continue-on-error: true
+      - name: Keep action running to view traces
+        if: steps.check-label.outputs.trace == 'true' && matrix.product.name == 'ocr2' && matrix.image.tag-suffix == '-plugins'
```

Comment:
Good idea! I do want to tinker with this since it requires the job to live for as long as traces need to be reviewed, but it's out of scope for this PR.